tests/manage/mcg/conftest.py

```diff
@@ -145,3 +150,38 @@ def awscli_pod(mcg_obj, created_pods):
     helpers.wait_for_resource_state(awscli_pod_obj, constants.STATUS_RUNNING)
     created_pods.append(awscli_pod_obj)
     return awscli_pod_obj
+
+
+@pytest.fixture()
+def mcg_storageclass(request):
```

Comment:
the same, try to utilize the storage class factory fixture we have

Reply:
Similary, storage class factory calls ceph_pool_factory_class and secret_factory_class which is not required for mcg sc unlike rbd and cephfs sc.